Scuti/Scripts/UI/Account/UserCreateCredentialForm.cs

```diff
@@ -0,0 +1,151 @@
+﻿using UnityEngine.UI;
```

Comment:
I don't think we want a new class here. I think we want to Update UserCredentialsForm instead.

Reply:
I will make the change.